app/org/sagebionetworks/bridge/stormpath/StormpathAccountDao.java

```diff
@@ -492,12 +508,24 @@ private Account constructAccount(StudyIdentifier studyId, com.stormpath.sdk.acco
         return account;
     }
     
-    private Account getAccountWithEmail(Study study, String email) {
+    public String getAccountIdFromEmail(Study study, String email) {
+        Directory directory = client.getResource(study.getStormpathHref(), Directory.class);
```

Comment:
This is not used? It's also untested and has a lot of redundancy with getAccountFromEmail(). One of these should call the other, or they should both call the same base method.